### PR TITLE
docs: adds backstage-plugin-ldap-auth

### PR DESCRIPTION
**Pull request by simonecorsi**
Signed-off-by: Simone Corsi <simonecorsi.dev@gmail.com>

## Hey, I just made a Pull Request!

We've developed a plugin to add LDAP authentication and think it can be useful to add to the marketplace!

I have just one question, we have two packages one for the FE and one for the BE, do I have to add two different entries or should I just add one with a `npmPackageName` pointing to one of it and then the user can get the other from the documentation?

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/data/plugins/backstage-plugin-ldap-auth.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Backstage Plugin LDAP Auth
```
Comment (jhaals) [inline microsite/data/plugins/backstage-plugin-ldap-auth.yml:2]:
```suggestion
title: LDAP Auth
```

File: microsite/data/plugins/backstage-plugin-ldap-auth.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+author: ImmobiliareLabs
+authorUrl: https://github.com/immobiliare
+category: Authentication
+description: Backstage plugin to authenticate users to an external LDAP server
```
Comment (jhaals) [inline microsite/data/plugins/backstage-plugin-ldap-auth.yml:6]:
```suggestion
description: Authenticate users to an external LDAP server
```

**Review by jhaals — CHANGES_REQUESTED** [review]
Thanks for the PR for the new plugin, some minor tweaks but good looking!
Yeah it's currently just one package and we typically reference the frontend package and integrators will have to discover the other package while reading the docs :)

File: microsite/data/plugins/backstage-plugin-ldap-auth.yml (hunk 1 above)
Comment (simonecorsi) [reply microsite/data/plugins/backstage-plugin-ldap-auth.yml]: Done!

File: microsite/data/plugins/backstage-plugin-ldap-auth.yml (hunk 2 above)
Comment (simonecorsi) [reply microsite/data/plugins/backstage-plugin-ldap-auth.yml]: Done!

**Comment by simonecorsi** [comment]
Ok, thanks for the answer! Let me know when we're done if you want me to squash into one commit!

**Review by jhaals — APPROVED** [review]
Thank you 👍

**Pull request merged**